packages/core-components/src/components/TopBar.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+export interface TopBarItemType {
+  className: string;
+  linkContact1: string;
+  linkContact2: string;
+  phoneNumber1: string;
+  phoneNumber2: string;
+  nameContact1: string;
+  nameContact2: string;
+}
+
+export interface TopBarProps {
+  topBarItems: TopBarItemType[];
+  brandIcon: React.ReactNode;
+  phoneIcon: React.ReactNode;
+  languageButton: React.ReactNode;
+}
+
+const TopBar: React.FC<TopBarProps> = ({
+  topBarItems,
+  phoneIcon,
+  languageButton,
+}) => {
+  const itemsProps = topBarItems[0];
```

Comment:
why do we have this topBarItems as a list only to get 1 single item ❓ 
the prop should either be a list with only 1 link/phone/name contact as its item, or a single `TopBarItemType`